Feature/kas 3358 Remove approval panel

TaneeG opened the pull request:
jira: https://kanselarij.atlassian.net/browse/KAS-3358

TaneeG committed:
KAS-3312 Octane refactor approval & panel

TaneeG committed:
KAS-3358 octane refactor fixing some errors

TaneeG committed:
Merge branch 'development' into feature/KAS-3312_refactor_approval_panel

TaneeG committed:
Merge branch 'feature/KAS-3312_refactor_procedurestap_componenten' into feature/KAS-3312_refactor_approval_panel

TaneeG committed:
Merge branch 'feature/KAS-3312_refactor_procedurestap_componenten' into feature/KAS-3312_refactor_approval_panel

ValenberghsSven commented:
Er zijn niet echt cypress testen hiervoor, ga dit nog manueel moeten testen.
Niet zeker of deze feature actief gebruikt wordt.

ValenberghsSven commented:
Werkt niet super efficient.
Je moet regelmatig refreshen om de veranderingen te zien. (ik vermoed dat dit ook al was pre refactor)

vb: 
- 2 ministers toevoegen > de panel toont met loader dat blijft staan. pas na refresh tonen de ministers
- akkoord zetten en opslaan > het vinkje komt er niet, pas na refresh.
- nieuwe minister toevoegen > de akkoorden zouden moeten gewist zijn (ergens anders) en de nieuwe minister moet erbij komen. pas na refresh is dit ok.


Zou het beter zijn om deze panel voorlopig uit onze app te halen? 
Deze wordt al jaren niet gebruikt op prod (0 resultaten op type "ext:Goedkeuring" op PROD)

erikap committed:
Remove agendaitems-approval-panel component

Since it's a dead feature not used in production legacy code can
better be removed from the codebase

erikap committed:
Remove approvals from data model

erikap commented:
As discussed on chat approvals are an unused feature adding complexity to the codebase. Legacy code/data model is therefore removed.

Backend PR: https://github.com/kanselarij-vlaanderen/app-kaleidos/pull/277

Custom Jenkins run:  http://kal-kastaar.s.redpencil.io:8080/job/kaleidos/job/custom_frontend_backend_combination/206/

erikap commented:
Custom Jenkins build succeeded.
